### PR TITLE
Feature/temporal wdpa

### PR DESCRIPTION
**Pull request by WKocur**

### CODE REVIEW

File: app/controllers/historic_wdpa_controller.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class HistoricWdpaController < ApplicationController
+  def index
+    wdpa_releases = HistoricWdpaRelease.all.order(:year, :month).as_json
+
+    wdpa_releases.each do |release|
+      release[:name] = "#{release['year']} - #{Date::MONTHNAMES[release['month']]}"
```
Comment (lucacug) [inline app/controllers/historic_wdpa_controller.rb]: It looks very good, I would only change this to `release['name']` for consistency.

**Review by lucacug — COMMENTED** [review]
@WKocur  I can't see the PR on the PP db repo related to the migration you have added

**Comment by WKocur** [comment]
> @WKocur I can't see the PR on the PP db repo related to the migration you have added

Oops, because there wasn't one! I've made one now. I'm assuming it should be with develop...

File: app/controllers/historic_wdpa_controller.rb (hunk 1 above)
Comment (WKocur) [reply app/controllers/historic_wdpa_controller.rb]: Cheers - will change that now :)

**Review by lucacug — COMMENTED** [review]
Looks good now from the BE side. Happy to merge if @stacytalbot has already checked the FE bits...

**Pull request closed without merging**